modules/plugin/imageio-ext-gdal/pom.xml

```diff
@@ -150,6 +150,12 @@
       <groupId>it.geosolutions.imageio-ext</groupId>
       <artifactId>imageio-ext-gdalvrt</artifactId>
       <!-- The version number is specified in the parent POM. -->
+      <exclusions>
+        <exclusion>
+          <groupId>it.geosolutions.imageio-ext</groupId>
+          <artifactId>imageio-ext-gdalgeotiff</artifactId>
+        </exclusion>
+      </exclusions>
```

Comment:
Why the exclusion?
Does the gdalvrt module depend on gdalgeotiff?
If that's the case we can consider opening also a ticket in imageio-ext and make sure to avoid that dependency.
Probably it has been added as a dependency for tests, without <scope>test</scope>